src/templates/QuickstartDetails.js

```diff
@@ -109,17 +109,39 @@ const QuickstartDetails = ({ data, location }) => {
             css={css`
               border-bottom: none;
               display: grid;
-              padding-bottom: 0;
-              grid-template-areas: 'title logo' 'summ logo' 'cta logo';
               grid-column-gap: 1rem;
               grid-row-gap: 1rem;
+              grid-template-areas:
+                'title logo'
+                'summ logo'
+                'cta logo';
+              justify-content: normal;
+              justify-self: center;
               row-gap: 1rem;
+              width: 101%;
```

Comment:
Setting the width to _more_ than `100%` is kinda weird, but I totally see why you're doing it. I tried playing around with the page layout to try and get the underlying content to be properly hidden underneath the header and couldn't come up with anything.

It would be nice to circle back to this at some point, but I don't think we should hold up this PR.

Reply:
yea if im remembering this correctly, it was just a pixel of the left most point of the W character that wasn't being hidden 🙈